procedures/release-duty/merge-duty/merge_duty.rst

```diff
@@ -376,31 +376,30 @@ Update the releng changelog
 Bump Nightly version and release dates in ShipIt
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-ShipIt currently hard-codes the version of Nightly that's being released, as
-well as the release dates.
+In ShipIt, the Firefox nightly version and the release dates are hard-coded, and a human must update them when a new nightly builds becomes available.
+The nightly builds trigger on a schedule at 10:00 and 22:00 hours UTC and take a few hours to complete.
+Once the new nightly version appears in `firefox/nightly/latest-mozilla-central/ <https://archive.mozilla.org/pub/firefox/nightly/latest-mozilla-central/>`__, we can update the nightly version and release dates in ShipIt.
+**Bumping the Firefox nightly version and release dates in ShipIt should be done on release day.**
+This way, we can be confident that the first nightly builds with the new version will be available to ShipIt.
 
-It doesn't get automatically updated because it would need to know when a new
-nightly was available, not just when the version had been updated in-tree.
-Everything up to merging this pull request can be done early, but the PR must
-not be merged before the first nightly has been built and published with the
-new version. However, it does need to be merged and deployed to `production <https://github.com/mozilla-releng/shipit#production>` 
-as soon as reasonably possible after nightly is built. Note: the product details API needs to be manually updated by pressing the gear icon in shipit
+**WARNING: Do not bump the nightly version or release dates in ShipIt until the new nightly builds are available.**
 
+These are the steps to bump the nightly version and release dates in ShipIt:
 1. ``git clone git@github.com:mozilla-releng/shipit.git``
 2. ``git checkout -b nightly_version_bump_${version}``
 3. Edit FIREFOX_NIGHTLY's major version in
-   https://github.com/mozilla-releng/shipit/blob/f3d45d1dd1cc08cc466865f7d39305f1b2edbcf7/api/src/shipit_api/common/config.py#L49
+   https://github.com/mozilla-releng/shipit/blob/main/api/src/shipit_api/common/config.py#L49
```

Comment:
The downside of pinning to a revision is the file will go out of date.
The downside of *not* pinning to a revision is the line number will go out of date. It may be worth re-pinning to revision.